scene/gui/tree.cpp

```diff
@@ -2449,13 +2449,13 @@ void Tree::_gui_input(Ref<InputEvent> p_event) {
 		}
 	}
 
-  if (drag_exited) {
+	if (drag_exited) {
```

Comment:
What i dont understand is why this code cant be in NOTIFICATION_WM_FOCUS out, did you try it and it didnt work?